scheme/src/EntityGeneratorBundle/Tools/AbstractEntityGenerator.php

```diff
@@ -1177,15 +1177,56 @@ protected function generateEntityAssociationMappingProperties(ClassMetadataInfo
                 continue;
             }
 
-            $lines[] = $this->generateAssociationMappingPropertyDocBlock($associationMapping, $metadata);
+            $doc = $this->generateAssociationMappingPropertyDocBlock($associationMapping, $metadata);
+            $lines[] = $doc;
```

Comment:
Is there a reason to use a temporal variable `$doc` here?

Reply:
Removed